lib/plugins.js

```diff
@@ -23,25 +23,25 @@ const libVideo = require('../plugins/video/lib/index');
 const pkg = require('../package');
 
 const plugins = [
-  { register: inert },
-  { register: vision },
-  { register: libAdmin, routes: { prefix: '/admin' } },
-  { register: viewAlbum, routes: { prefix: '/view' } },
-  { register: editAlbum, routes: { prefix: '/edit' } },
-  { register: exploreVideo, routes: { prefix: '/explore' } },
-  { register: libGallery },
-  { register: libGeojson, routes: { prefix: '/geojson' } },
-  { register: libHome },
-  { register: libPublic },
-  { register: libRename, routes: { prefix: '/admin' } },
-  { register: libResize, routes: { prefix: '/admin' } },
-  { register: libVideo, routes: { prefix: '/view' } },
-  { register: libWalk },
+  inert,
+  vision,
+  { plugin: libAdmin, routes: { prefix: '/admin' } },
+  { plugin: viewAlbum, routes: { prefix: '/view' } },
+  { plugin: editAlbum, routes: { prefix: '/edit' } },
+  { plugin: exploreVideo, routes: { prefix: '/explore' } },
+  { plugin: libGallery },
+  { plugin: libGeojson, routes: { prefix: '/geojson' } },
+  { plugin: libHome },
+  { plugin: libPublic },
+  { plugin: libRename, routes: { prefix: '/admin' } },
+  { plugin: libResize, routes: { prefix: '/admin' } },
+  { plugin: libVideo, routes: { prefix: '/view' } },
+  { plugin: libWalk },
   {
-    register: hapiSwagger,
+    plugin: hapiSwagger,
     options: { info: { title: 'history API', version: pkg.version } }
-  },
-  { register: lout }
+  }
+  // { plugin: lout } doesn't support hapi 17 yet
```

Comment:
Waiting on this [lout plugin support issue](https://github.com/hapijs/lout/issues/175)